warrior/Warriorspace/wrapper_files/tw_sample.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" ?>
```

Comment:
The same is file is added from 2041 commit also.

Reply:
Removed from 2041 branch.